app/helpers/markdown_helper.rb

```diff
@@ -0,0 +1,12 @@
+module MarkdownHelper
+  class << self
+    def markdown(text)
+      unless @markdown
+        renderer = Redcarpet::Render::HTML.new(filter_html: true, hard_wrap: true)
+        @markdown = Redcarpet::Markdown.new(renderer, tables: true)
+      end
+
+      @markdown.render(text).html_safe
```

Comment:
__[RuboCop]__

Rails/OutputSafety: Tagging a string as html safe may be a security risk.


<!-- Comment by [SideCI](https://sideci.com) -->